loop/index.js

```diff
@@ -42,6 +42,8 @@ var storage = getStorage(conf.get("storage"), {
 var tokBox = new TokBox(conf.get('tokBox'));
 
 var ravenClient = new raven.Client(conf.get('sentryDSN'));
+ravenClient.captureError(new Error('Server could communicate with Sentry'));
```

Comment:
Since this is not an error, we could use a different level than the default one for this message.

We could use the `info` level instead, as described at https://github.com/getsentry/raven-node#basic-usage
Though, this means we need to create another raven client and configure it differently somehow (just with a different parameter)

What do you think?